src/garage/torch/algos/vpg.py

```diff
@@ -107,34 +123,28 @@ def _check_entropy_configuration(self, entropy_method, center_adv,
 
     def train_once(self, itr, paths):
         """Perform one step of policy optimization."""
-        samples_data = self.process_samples(itr, paths)
+        valids, obs, actions, rewards, average_return = self.process_samples(
+            itr, paths)  # noqa: E501
```

Comment:
why is there a noqa here? the line isn't too long.

Reply:
removed